From d432ed5559ff467f5ef1e40ce926b88942f5eb47 Mon Sep 17 00:00:00 2001
From: jbf-insidesherpa <54832297+jbf-insidesherpa@users.noreply.github.com>
Date: Thu, 31 Oct 2019 15:18:18 +0800
Subject: [PATCH 04/11] Update README

- copy similar updates on READ of python 2 version of repo
---
 README.markdown | 30 ++++++++++++++++--------------
 1 file changed, 16 insertions(+), 14 deletions(-)

diff --git a/README.markdown b/README.markdown
index 5fc8f5b..d648444 100644
--- a/README.markdown
+++ b/README.markdown
@@ -24,21 +24,18 @@
 
 <ol>
 	<li>Please clone this repository to start the task</li>
-	<li>Adjust the getRatio and getDataPoint functions</li>
-	<li>Pass all unit tests and add more to cover edge cases</li>
+	<li>Adjust the getRatio, getDataPoint and main functions</li>
+	<li>Bonus: Pass all unit tests and add more to cover edge cases</li>
 	<li>Upload a git patch file as the submission to this task</li>
 	
 </ol>
 
-<h2 id="installation" >Installation</h2>
+<h2 id="installation" >Set up / Installation</h2>
 
-Please ensure you are using <b> python3. </b>
+<p>In order to get the server and client application code working on your machine, <a href="https://insidesherpa.s3.amazonaws.com/vinternships/companyassets/Sj7temL583QAYpHXD/setup_devenv_m1_v2.pdf">follow the setup here</a></p>
+<p><b>Note:</b>This is the Python 3 version of the JPM 1 exercise. The Python 2.7 version is in <a href="https://github.com/insidesherpa/JPMC-tech-task-1">this other repo</a></p>
 
-Start the data feed server by running:
-
-<code> python server.py</code>
-
-<h2>Run</h2>
+<h2>How to Run</h2>
 To start the server, run
 
 	python server3.py
@@ -49,7 +46,7 @@ If you encounter an issue with `datautil.parser`, run this command:
 
 	pip install python-dateutil
 
-If you don't have pip, you can install it from: https://pip.pypa.io/en/stable/installing/
+If you don't have pip yet, you can install it from: https://pip.pypa.io/en/stable/installing/
 
 To start the example client, run:
 
@@ -58,11 +55,16 @@ To start the example client, run:
 To unit test the example client, run:
 	python client_test.py
 
-<h2>API Examples</h2>
-
-See also [client.py](https://github.com/texodus/exchange_simulator/blob/master/client.py)
-
+<h2>How to request from the server using curl</h2>
+<!--See also [client.py](https://github.com/texodus/exchange_simulator/blob/master/client.py)-->
 Query:
 
 	$ curl 'http://localhost:8080/query?id=1'
 	{"id": "1", "top_ask": {"price": 129.18, "size": 70}, "timestamp": "2016-08-06 12:32:11.821574", "top_bid": {"price": 128.79, "size": 61}}
+
+<h2>How to fix the code to meet objectives</h2>
+<p>To make the changes necessary to complete the objectives of this task, <a href="https://insidesherpa.s3.amazonaws.com/vinternships/companyassets/Sj7temL583QAYpHXD/making_changes_m1_v2.pdf">follow this guide</a>.</p>
+<p>To do the bonus task, <a href="https://insidesherpa.s3.amazonaws.com/vinternships/companyassets/Sj7temL583QAYpHXD/Client%20Test%20Part.pdf">read this</a>.</p>
+
+<h2>How to submit your work</h2>
+<p>A patch file is what is required from you to submit. To create a patch file, <a href="https://insidesherpa.s3.amazonaws.com/vinternships/companyassets/Sj7temL583QAYpHXD/create_patch_file_v2.pdf">follow this guide</a>. Then submit the patch file in the <a href="https://www.insidesherpa.com/modules/R5iK7HMxJGBgaSbvk/gtAhtcvke9AFCzqME">JPM Module 1 Page</a>.</p>
-- 
2.25.0.windows.1

